tests/test_gpu_examples.py

```diff
@@ -1550,6 +1550,98 @@ def on_optimizer_step(self, args, state, control, **kwargs):
             # assert loss is not None
             assert trainer.state.log_history[-1]["train_loss"] is not None
 
+    @pytest.mark.single_gpu_tests
```

Comment:
You put this test into wrong class, which results in it trying to load a GPTQ quantized base model. Please put it in the previous test class at line ~1393.